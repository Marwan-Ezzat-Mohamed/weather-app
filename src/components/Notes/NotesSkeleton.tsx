const NotesSkeleton = () => (
  <div className="w-full space-y-2 bg-slate-100 dark:bg-slate-800">
    <div className="h-16 w-full animate-pulse rounded-md bg-gray-300 dark:bg-gray-500"></div>
    <div className="h-16 w-full animate-pulse rounded-md bg-gray-300 dark:bg-gray-500"></div>
    <div className="h-16 w-full animate-pulse rounded-md bg-gray-300 dark:bg-gray-500"></div>
  </div>
);

export { NotesSkeleton };
