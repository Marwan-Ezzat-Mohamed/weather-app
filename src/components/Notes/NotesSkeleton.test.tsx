import { render } from "@testing-library/react";
import { NotesSkeleton } from "./NotesSkeleton";

test("WeatherCardSkeleton matches snapshot", () => {
  const { container } = render(<NotesSkeleton />);
  expect(container).toMatchSnapshot();
});
