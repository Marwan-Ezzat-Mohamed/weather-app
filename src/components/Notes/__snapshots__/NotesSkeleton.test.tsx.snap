// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeatherCardSkeleton matches snapshot 1`] = `
<div>
  <div
    class="w-full space-y-2 bg-slate-100 dark:bg-slate-800"
  >
    <div
      class="h-16 w-full animate-pulse rounded-md bg-gray-300 dark:bg-gray-500"
    />
    <div
      class="h-16 w-full animate-pulse rounded-md bg-gray-300 dark:bg-gray-500"
    />
    <div
      class="h-16 w-full animate-pulse rounded-md bg-gray-300 dark:bg-gray-500"
    />
  </div>
</div>
`;
